docs/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 ### Changed
 - Removed `last_validated_dt` from exposed JSON Flaw History data (OSIDB-3814)
+- Trim flaw summary if it is too long (OSIDB-3847)
```

Comment:
I do not think that this is a good description. The flaw does not have a **summary** attribute so the user reading this would hardly understand. We are actually not even trimming any flaw attribute. The entity in question is the Jira task